src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -61,8 +77,10 @@ const newRule = {
   calculation: '',
   timePeriod: '',
   conditions: [newCondition()], // Start with one condition
+  actions: [newAction()], // Start with one action
   severity: Config.ruleSeverity.critical,
-  enabled: true
+  enabled: true,
```

Comment:
can this be named `ruleEnabled `instead of `enabled` it's confusing.

Reply:
I think it makes sense, it looks like newRule.enabled. newRule.ruleEnabled is redundant (also this was already there)